test/contract/mappings/StringStartsWithCapital.json

```diff
@@ -7,7 +7,13 @@
     "status": 200,
     "jsonBody": {
       "search_string": "Totnes",
-      "number_of_services": 1,
+      "number_of_services_found": 1,
```

Comment:
This JSON isn't quite matching the schema.  I've updated and pushed changes to the address.json mapping file so you can see how it is meant to be formatted.  Can you bring the others in line please?